service/src/main/java/org/cbioportal/genome_nexus/service/internal/VariantAnnotationSummaryServiceImpl.java

```diff
@@ -72,6 +78,10 @@ public VariantAnnotationSummaryServiceImpl(VariantAnnotationService hgvsVariantA
         this.variantClassificationResolver = variantClassificationResolver;
         this.variantTypeResolver = variantTypeResolver;
         this.exonResolver = exonResolver;
+
+        this.canonicalTranscriptResolver.setPredefinedCanonicalTranscriptIds(
+            this.ensemblService.getCanonicalTranscriptIdsBySource(IsoformOverrideSource.GENOME_NEXUS)
+        );
```

Comment:
Let's try to figure out what does should be. Ideally the annotation summary service should use the `isoformOverrideSource`